src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -1243,3 +1243,13 @@ public int IntValue
         }
     }
 }
+
+public class TestSerialize
+{
```

Comment:
I would like to see an regular element in the class too. It's the difference between
```xml
<TestSerialize Attr="1"/>
```
and
```xml
<TestSerialize Attr="1">
  <MyInt>7</MyInt>
</TestSerialize>
```